docs/how/delete-metadata.md

```diff
@@ -2,11 +2,23 @@
 
 There are a two ways to delete data from DataHub.
 
+
```

Comment:
Would love a dedicated CLI doc, that this doc simply points to :)

Reply:
Yes!! I dont want to expand the scope of this PR, but completely agree.